enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -1147,7 +1150,8 @@ func (p *PebbleCache) deleteOrphanedFiles(quitChan chan struct{}) error {
 		}
 
 		unlockFn := p.locker.RLock(key.LockID())
-		_, err = p.lookupFileMetadata(p.env.GetServerContext(), iter, key)
+		md, err := p.lookupFileMetadata(p.env.GetServerContext(), iter, key)
+		md.ReturnToVTPool()
```

Comment:
Having to remember to call `ReturnToVTPool` on the type returned from `lookupFileMetadata` feels a little error-prone to me. Would it be safer if the caller was required to pass the output proto struct to `lookupFileMetadata`?